www/shared/i18n/en.i18n.json

```diff
@@ -29,5 +29,8 @@
   "Updating_files": "Updating files",
   "Updating_files_s_s": "Updating files<br/>%s / %s",
   "Validating_server": "Validating server",
-  "Warning": "Warning"
+  "Warning": "Warning",
+  "new_native_apps_title": "Our new native apps are available",
+  "new_native_apps_description": "They have a much better performance, give it a try!",
```

Comment:
They have been completely rebuilt from the ground up for performance, and a better user experience.  Give it a try!